app-utils/date-utils.js

```diff
@@ -0,0 +1,25 @@
+const getPreviousWeekDates = () => {
+  const currentDate = new Date();
+
+  const mondayOfPreviousWeek = new Date(currentDate);
```

Comment:
current date is always 'now' (new Date()), so it is not necessary to create variables in that way - actually, it's `new Date(new Date())`